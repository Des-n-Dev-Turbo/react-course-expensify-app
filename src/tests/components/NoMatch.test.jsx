import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NoMatch from '../../components/NoMatch';

test('Should render error page', () => {
    const wrapper = shallow(<NoMatch />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});