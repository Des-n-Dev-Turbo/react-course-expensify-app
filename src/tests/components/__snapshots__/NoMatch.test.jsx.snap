// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render error page 1`] = `
<div>
  <h1>
    404!
  </h1>
</div>
`;
